Updated README.md

shashank136 opened the pull request:
## Acceptance Critera
- [x] Add wiki links to README.
- [x] Add Sprint1.md link to README. Refer #35 . 
- [x]  Move README content to a "Home/About" page in wiki.

shashank136 committed:
Update README.md

Updated the readme file to add links for accessing documentation.

haxxorsid commented:
@shashank136 I have moved everything in README to Home page of wiki. You can clear everything in README, except group member names  and add all 4-5 Wiki page links.

Once, we have a Sprint1.md, we should link that as well from README.

shashank136 committed:
Updated the readme to remove redundant information

shashank136 commented:
> @shashank136 I have moved everything in README to Home page of wiki. You can clear everything in README, except group member names and add all 4 Wiki page links (you left out Home page).
> 
> Once, we have a Sprint1.md, we should link that as well from README.

Updated the readme file to direct reader to Wiki pages

haxxorsid commented on README.md:
```diff
@@ -7,46 +7,13 @@ A referral network to get referral from verified professionals.
+- [API Reference](https://github.com/haxxorsid/referralboard/wiki/API-Reference)
+- [front-end Documentation](https://github.com/haxxorsid/referralboard/wiki/Frontend-Documentation)
 
+### How to raise an bug or feature request?
```
"How to raise an bug or feature request?" is about creating an issue. Our branching strategy only talks about creating Pull requests and branches. So, you can write "Branching Strategy" or "Git Branching Strategy" instead.

haxxorsid commented on README.md:
```diff
@@ -7,46 +7,13 @@ A referral network to get referral from verified professionals.
-- Go
-- React (Typescript)
-- Postgres 14.1.
+### About the project
```
Need better heading.

shashank136 replied:
Suggestions are welcome 😅

haxxorsid replied:
Wiki or Getting Started or Documentation or Learn More.

haxxorsid replied:
Wiki or Getting Started or Documentation or Learn More or Index

shashank136 committed:
updated about GIT Branching Strategy

haxxorsid commented on README.md:
```diff
@@ -7,46 +7,13 @@ A referral network to get referral from verified professionals.
+- [API Reference](https://github.com/haxxorsid/referralboard/wiki/API-Reference)
+- [front-end Documentation](https://github.com/haxxorsid/referralboard/wiki/Frontend-Documentation)
 
+### GIT Branching Strategy?
```
remove question mark

shashank136 committed:
Update README.md

shashank136 committed:
Update README.md

haxxorsid commented on README.md:
```diff
@@ -7,46 +7,12 @@ A referral network to get referral from verified professionals.
-- React (Typescript)
-- Postgres 14.1.
+### Wiki
+- [Wiki](https://github.com/haxxorsid/referralboard/wiki)
```
Line 11 About or About project or About the project?

shashank136 committed:
Update README.md

haxxorsid commented on README.md:
```diff
@@ -7,46 +7,12 @@ A referral network to get referral from verified professionals.
-- React (Typescript)
-- Postgres 14.1.
+### Wiki
+- [About the project?](https://github.com/haxxorsid/referralboard/wiki)
```
![Drag Racing](https://i.ibb.co/Lxzs5H1/63w8qv.jpg)

haxxorsid commented on README.md:
```diff
@@ -7,46 +7,12 @@ A referral network to get referral from verified professionals.
-- Go
-- React (Typescript)
-- Postgres 14.1.
+### Wiki
```
##Wiki not ###Wiki

shashank136 committed:
DONE !!!!!!

DONE !!!!!!

shashank136 committed:
Update README.md

haxxorsid committed:
Updated README.md

haxxorsid reviewed (dismissed):
Approved

haxxorsid committed:
Update README.md

Vineetkhadloya reviewed (dismissed):
Good.

haxxorsid committed:
Delete README.md

haxxorsid committed:
Update README.md

haxxorsid committed:
Update README.md

haxxorsid committed:
Update README.md

haxxorsid committed:
Merge remote-tracking branch 'origin/main' into feature/updateforreadme

haxxorsid committed:
Delete user-story-template.md

Vineetkhadloya reviewed (approved):
Good Job !!
